.github/workflows/lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: ESLint
+on:
+    push:
+    pull_request:
+    schedule:
+        - cron: '0 */12 * * *'
```

Comment:
Running eslint on a schedule is useless. New eslint violations won't ever be introduced without a push.

Reply:
Good point - my head was over in MakerBundle where we run twice a day to catch upstream releases that break things